locale/bg.po

```diff
@@ -1991,6 +1991,11 @@ msgstr "Външност"
 msgid "BEHAVIOR"
 msgstr "Поведение"
 
+#: ../src/microbe_stage/editor/MicrobeEditor.tscn:1568
+#, fuzzy
+msgid "SEARCH_DOT_DOT_DOT"
+msgstr "..."
```

Comment:
The text for this used to be `Search...`

Reply:
That's for Bulgarian? He put the correct translation for english

Reply:
This is the bg translation, that was automatically generated by gettext (note the `#, fuzzy` part, which is added by gettext when it tries to guess a potential translation).

Reply:
Oh huh, didn't see that woops, alright this should be good then.